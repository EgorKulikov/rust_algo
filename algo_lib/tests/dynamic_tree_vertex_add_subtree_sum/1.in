5 7
1 10 100 1000 10000
0 1
1 2
2 3
1 4
2 1 2
1 1 100000
2 1 2
0 1 2 2 0
2 0 2
0 2 3 3 1
2 1 4