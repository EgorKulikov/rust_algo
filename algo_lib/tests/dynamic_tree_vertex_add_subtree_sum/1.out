10011
110011
110011
101111